App/Stylesheets/index.css

```diff
@@ -21,4 +19,54 @@ p#titleParagraph {
     padding: 0;
     color: brown;
     font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
-  }
\ No newline at end of file
+    text-align: Center;
+  }
+
+  #title {
+    height: 25%;
+    background-color: var(--primary-bg);
```

Comment:
There is no `--primary-bg` variable in the file since you deleted it from the `:root` section. Please put it back, or change this `background-color` rule.